keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -517,8 +594,10 @@ enabled even if the annotation has been set on the Service.
   with before the feature was enabled.
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  This feature is not yet implemented but per-Service enablement/disablement is
-  covered in depth as part of the test plan.
+  Per-Service enablement/disablement is covered in depth and feature gate
+  enablement and disablement will be covered before the feature graduates to GA.
+  In addition, manual testing covering combinations of
+  upgrade->downgrade->upgrade cycles will be completed prior to GA graduation.
```

Comment:
Too late now, but we should have done that before Beta (which enabled that by default).
Anyway - let's that that testing before graduating to GA and update the KEP with the results.

Reply:
Makes sense, will do. I'm pretty sure I did this kind of manual testing, but it was more than a year ago and I clearly failed to record any results in the GEP.